app/reducers/quotes.js

```diff
@@ -91,6 +91,17 @@ export const selectQuotes = createSelector(
     )
 );
 
+export const selectQuotesByApproval = createSelector(
+  (state) => state.quotes.byId,
+  (state) => state.quotes.items,
+  (state, props) => props?.pagination,
+  (state, props) => props?.approvalStatus,
+  (quotesById, items, pagination, approvalStatus) =>
+    (pagination ? pagination.items : items)
+      .map((quoteId) => quotesById[quoteId])
+      .filter((quote) => quote.approved === approvalStatus)
```

Comment:
I believe this is done correctly, but it can be simplified by using the existing `selectQuotes`-selector :)

```suggestion
  selectQuotes,
  (_, props) => props?.approvalStatus,
  (quotes, approvalStatus) =>
    quotes.filter((quote) => quote.approved === approvalStatus)
```

Reply:
Agreed